File: pyg_interface/src/main/java/com/itheima/pyg/service/user/AddressService.java
package com.itheima.pyg.service.user;

import com.itheima.pyg.pojo.address.Address;

import java.util.List;

public interface AddressService {
    List<Address>   findListByLoginUser(String userId);

    //查询用户收货地址
    public List<Address>findListByUserName(String userName);

    //新增用户收货地址
    public void addAddress(Address address, String userId);

    //删除用户收货地址
    public void deleteAddress(Long id);

    //修改用户收货地址

    public Address updateAddrss(Long id);

    //编辑收货用户地址
    public void editorAddress(Long id);
}

File: pyg_web_user/src/main/webapp/js/service/addressService.js
//地址服务层
app.service('addressService',function ($http) {
    //获取收货列表
    this.findListByUserName=function () {
        return $http.get("address/findListByUserName.action")
    };


    //添加地址信息
    this.addAddress=function (entity) {
        return $http.post("/address/addAddress.action?",entity);
    };


    //删除地址信息
    this.delete=function (id) {
        return $http.post("/address/delete.action?id="+id);
    };

    //修改
    this.update=function (id) {
        return $http.post("/address/update.action?id="+id);
    };

    //编辑地址信息
    this.editorAddress=function (id) {
        return $http.post("/address/editorAddress.action?id="+id);
    };
});